web/text_layer_builder.js

```diff
@@ -212,6 +221,19 @@ class TextLayerBuilder {
       }
       end.classList.remove("active");
     });
+
+    div.addEventListener("copy", event => {
+      const selection = document.getSelection();
+      event.clipboardData.setData(
+        "text/plain",
+        selection
+          .toString()
+          .replaceAll(TextLayerBuilder.NormalizeRegex, m =>
+            m[0].normalize("NFKC")
+          )
```

Comment:
Something that I just realized is that we need to remove null chars here as well, since copying otherwise won't work correctly in all cases; note what happens when selecting and copying the contents of "Table 3.15" from this page: https://ftp.acc.umu.se/mirror/CTAN/info/lshort/english/lshort.pdf#page=83
```suggestion
        removeNullCharacters(
          selection
            .toString()
            .replaceAll(TextLayerBuilder.NormalizeRegex, m =>
              m[0].normalize("NFKC")
            )
        )
```